web/views/pages/index.ejs

```diff
@@ -297,21 +301,21 @@
         </div>
         <div class="loading overlay"></div>
         <div id="embedded-error" class="overlay" style="background-color: grey; display: none">
-            Embedding of the ElectricityMap has been deactivated. You can still access it at <a href="http://www.electricitymap.org" target="_blank">http://www.electricitymap.org</a>.<br />
-            Please contact us at <a href="mailto:hello@tmrow.com">hello@tmrow.com</a> for more information.
+            <%= __('embedding.deactivated') %>. <%= __('embedding.stillaccess') %> <a href="http://www.electricitymap.org" target="_blank">http://www.electricitymap.org</a>.<br />
+            <%= __('embedding.contactus') %> <a href="mailto:hello@tmrow.com">hello@tmrow.com</a> <%= __('embedding.moreinformation') %>.
         </div>
         <div id="countrypanel-production-tooltip" class="tooltip panel">
-            <span id="production-proportion"></span> <%= __('tooltips.ofelectricity') %> <img id="country-flag" class="flag"></img> <span class="country-code"></span> <span id="domain-name"></span><br />
+            <%= __('tooltips.the') %> <span id="production-proportion"></span> <%= __('tooltips.ofelectricity') %> <img id="country-flag" class="flag"></img> <span class="country-code"></span> <span id="domain-name"></span><br />
```

Comment:
I'm not comfortable adding "the" because it really depends of languages.
I'm working on making a whole sentence in the translation system. I suggest removing `the` and you'll see that the next version should be easier to translate.